### PR TITLE
Remove gherkin-streams from the monorepo

### PR DESCRIPTION
**Pull request by aurelien-reeves**
## Summary

Removes the gherkin-streams from the monorepo as it is now located at [github.com/cucumber/gherkin-streams](https://github.com/cucumber/gherkin-streams)

### CODE REVIEW

File: fake-cucumber/javascript/.mocharc.local.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (davidjgoss) [inline fake-cucumber/javascript/.mocharc.local.json:1]: So this is the same as `.mocharc.local.json` but without the require of tsconfig-paths - does it break without this? Does it also mean you have to build dependent modules before running tests now?

Comment (davidjgoss) [reply fake-cucumber/javascript/.mocharc.local.json:1]: (Not the end of the world if so, as these will presumably be coming out of the monorepo soon enough, but curious.)

Comment (aurelien-reeves) [reply fake-cucumber/javascript/.mocharc.local.json:1]: Yes, tsconfig-paths does break the build.

Actually if I recall correctly,we already need to build JavaScript modules before anything else

**Review by davidjgoss — APPROVED** [review]

**Comment by aurelien-reeves** [comment]
Actually it seems tsconfig-paths was not required as part of the mocha configuration at least

**Pull request merged**